### DISCUSSION TITLE
Is the Documentation webpage (github-pages) meant to be down?

### DISCUSSION THREAD

**Discussion by TFWol** [oobabooga/text-generation-webui#2151] [Q&A]
I was able to fork the repo and enable the page fine on my end, so I'm just wondering if the help docs webpage is in an error state on purpose.

**Comment by BarfingLemurs** [comment]
no github-pages?

**Reply by TFWol** [reply]
I'm not sure what you mean, but...

For example, the page that's deployed in this repo is https://oobabooga.github.io/text-generation-webui/
It leads to a 404.

I cloned this repo and deployed it myself: https://tfwol.github.io/text-generation-webui/
This works.
